lynx/common/scene.py

```diff
@@ -63,3 +64,17 @@ def remove_object(self, object: Object) -> NoReturn:
 
     def add_to_pending_actions(self, action: str) -> NoReturn:
         self.pending_actions.append(action)
+
+    def get_walkable_positions(self):
+        walkable_positions = []
+        for position in self._square_position_map.keys():
+            if self._square_position_map[position].walkable():
+                walkable_positions.append(position)
+        return walkable_positions
+
+    def generate_drop_area(self, player: str):
+        walkable_positions = self.get_walkable_positions()
```

Comment:
Maybe, we can select a position on the map and then see if it can be walekd on, if not, randomly select the position again?
In addition, it is possible to get the drop area of all players and set the drop at least X position from another, to prevent players from having a drop area close to each other?

Reply:
Imo random value is better, and we don't need to add a lot on godot site.

Reply:
As discussed with @alpinus4, `generate_drop_area` is called so rarely (onlu upon new Player connecting to the Server), that we can afford this arguably cleaner solution